README.md

```diff
@@ -102,11 +102,50 @@ A preview of the VDK CLI commands:
 
 
 # Getting Started
-Installing VDK is a simple pip command. See the [Getting Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started) guide to install VDK and create a data job.
+
+## Create and run data jobs locally
+
+```
+pip quickstart-vdk
+```
+
+This installs the core vdk packages and the vdk command line interface. You can
+use them to run jobs in your local shell environment.
+
+See also the [Getting
+Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started)
+section of the wiki
+
+## Deploy and manage jobs in a cloud-like environment
+
+**Prerequisites**
+- [helm](https://helm.sh/docs/intro/install)
+- [docker](https://docs.docker.com/get-docker)
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) (version 0.11.1 or later)
+
+```
+vdk server --install
+```
+
+The server includes:
+- control service - provides an API for managing the lifecycle of Data Jobs and
+  an execution environment where jobs can be deployed
```

Comment:
I would very much like to see this sentence `provides an API for managing the lifecycle of Data Jobs[, ...for what purpose?]` -> expanded with stating the purpose of this API. For example, the benefits of using Kubernetes cloud in terms of continuous delivery, collaboration/teamwork, orchestration, high availability, security/isolation. May be a link to such a document.

Reply:
Added as the section overview instead.